ISSUE-3: Inconsistent Lint rules depending on test runner

Different lint standards are being applied for different ways to run `lint`:

1. `$ gulp lint`

```
[22:49:39] Using gulpfile ~/Documents/Projects/PSR-8/gulpfile.js
[22:49:39] Starting 'phpcs'...
[22:49:39] Starting 'phplint'...
[22:49:40] Finished 'phplint' after 1.25 s
[22:49:40] Finished 'phpcs' after 1.37 s
[22:49:40] Starting 'lint'...
[22:49:40] Finished 'lint' after 14 μs
```
2. `$ npm run-script lint`
```
$ npm run-script lint

> psr-8@0.1.0 lint /Users/darrenlee/Documents/Projects/PSR-8
> php ./vendor/bin/phpcs --colors src


FILE: /Users/darrenlee/Documents/Projects/PSR-8/src/GroupHuggable.php
----------------------------------------------------------------------
FOUND 14 ERRORS AND 1 WARNING AFFECTING 6 LINES
----------------------------------------------------------------------
  2 | ERROR   | Doc comment is empty
  4 | WARNING | PHP version not specified
  4 | ERROR   | Missing @category tag in file comment
  4 | ERROR   | Missing @package tag in file comment
  4 | ERROR   | Missing @author tag in file comment
  4 | ERROR   | Missing @license tag in file comment
  4 | ERROR   | Missing @link tag in file comment
 11 | ERROR   | Missing @category tag in class comment
 11 | ERROR   | Missing @package tag in class comment
 11 | ERROR   | Missing @author tag in class comment
 11 | ERROR   | Missing @license tag in class comment
 11 | ERROR   | Missing @link tag in class comment
 15 | ERROR   | Doc comment for parameter "$huggables" missing
 22 | ERROR   | Missing parameter name
 24 | ERROR   | Missing @return tag in function comment
----------------------------------------------------------------------


FILE: /Users/darrenlee/Documents/Projects/PSR-8/src/Huggable.php
----------------------------------------------------------------------
FOUND 14 ERRORS AND 1 WARNING AFFECTING 6 LINES
----------------------------------------------------------------------
  2 | ERROR   | [ ] Doc comment is empty
  4 | WARNING | [ ] PHP version not specified
  4 | ERROR   | [ ] Missing @category tag in file comment
  4 | ERROR   | [ ] Missing @package tag in file comment
  4 | ERROR   | [ ] Missing @author tag in file comment
  4 | ERROR   | [ ] Missing @license tag in file comment
  4 | ERROR   | [ ] Missing @link tag in file comment
 12 | ERROR   | [ ] Missing @category tag in class comment
 12 | ERROR   | [ ] Missing @package tag in class comment
 12 | ERROR   | [ ] Missing @author tag in class comment
 12 | ERROR   | [ ] Missing @license tag in class comment
 12 | ERROR   | [ ] Missing @link tag in class comment
 16 | ERROR   | [x] Line indented incorrectly; expected at least 4
    |         |     spaces, found 2
 23 | ERROR   | [ ] Missing parameter comment
 25 | ERROR   | [ ] Missing @return tag in function comment
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 44ms; Memory: 4Mb


npm ERR! Darwin 16.4.0
npm ERR! argv "/usr/local/Cellar/node/6.7.0/bin/node" "/usr/local/bin/npm" "run-script" "lint"
npm ERR! node v6.7.0
npm ERR! npm  v3.10.7
npm ERR! code ELIFECYCLE
npm ERR! psr-8@0.1.0 lint: `php ./vendor/bin/phpcs --colors src`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the psr-8@0.1.0 lint script 'php ./vendor/bin/phpcs --colors src'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the psr-8 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     php ./vendor/bin/phpcs --colors src
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs psr-8
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls psr-8
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/darrenlee/Documents/Projects/PSR-8/npm-debug.log
```

--- a/README.md
+++ b/README.md
@@ -24,7 +24,6 @@ $ php index.php
 or listing of all coding volations by file.
 
 - `$ php ./vendor/bin/phpcs --standard=PSR2 --colors -s index.php src tests`
-- `$ php ./vendor/bin/phpcs --standard=./phpunit.xml.dest --colors -s index.php src tests`
 
 or automate processing of files that will be adjusted to meet coding standards.
 

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -51,7 +51,7 @@ gulp.task('phplint', function (cb) {
  */
 gulp.task('phpunit', function () {
     var options = {debug: false, notify: true};
-    gulp.src('phpunit.xml')
+    gulp.src('phpunit.xml.dist')
         .pipe(phpunit('vendor/bin/phpunit --verbose tests', options))
         .on('error', notify.onError({
             title: "Failed Tests!",
@@ -72,7 +72,7 @@ gulp.task('phpcs', function () {
     return gulp.src(['./**/*.php', 'bin', '!./node_modules/', '!./vendor/**/*'])
         .pipe(phpcs({
             bin: 'vendor/bin/phpcs',
-            standard: 'phpunit.xml.dist',
+            standard: 'PSR2',
             warningSeverity: 0,
             showSniffCode: 1,
             colors: 1
@@ -88,12 +88,12 @@ gulp.task('phpcbf', shell.task(['vendor/bin/phpcbf --standard=PSR2 --ignore=vend
  * Watch functionality built in to base package. See entry in "default" command.
  */
 gulp.task('watch', function () {
-    gulp.watch(['composer.json', 'phpunit.xml', './**/*.php', '!./vendor/**/*', '!./node_modules/**/*'],
+    gulp.watch(['composer.json', 'phpunit.xml.dist', './**/*.php', '!./vendor/**/*', '!./node_modules/**/*'],
         function (event) {
             console.log('File ' + event.path + ' was ' + event.type + ', running tasks...');
         });
     gulp.watch('composer.json', ['dump-autoload']);
-    gulp.watch(['phpunit.xml', './**/*.php', '!vendor/**/*', '!node_modules/**/*'], ['phplint', 'phpunit']);
+    gulp.watch(['phpunit.xml.dest', './**/*.php', '!vendor/**/*', '!node_modules/**/*'], ['phplint', 'phpunit']);
 });
 
 

--- a/package.json
+++ b/package.json
@@ -20,6 +20,6 @@
   "scripts": {
     "start": "php index.php",
     "test": "php ./vendor/bin/phpunit --verbose tests",
-    "lint": "php ./vendor/bin/phpcs --colors src tests"
+    "lint": "php ./vendor/bin/phpcs --standard=PSR2 --colors src tests"
   }
 }

